source/php/Module/views/partials/card.blade.php

```diff
@@ -1,8 +1,9 @@
-<div class="o-grid-12@md">
+<div class="o-grid-{{ $gridColumns }}">
```

Comment:
This might be further elaborated. For example, 4 columns in desktop view, may be rendered as two columns in mobile view etc. 

An example of a similar implementation can be found here: https://github.com/helsingborg-stad/Municipio/blob/2314e5cf29ab766c2c8eaf0f2a816d36007d3e34/library/Controller/Archive.php#L126